external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceRDD.scala

```diff
@@ -19,18 +19,15 @@ package org.apache.spark.sql.kafka010
 
 import java.{util => ju}
 
-import scala.collection.mutable.ArrayBuffer
```

Comment:
I've also cleaned up unused imports and remove unnecessary line (two empty lines between the last import statement and class definition)